prototype/src/services/relationshipService.ts

```diff
@@ -0,0 +1,45 @@
+export interface Relationship {
+    id: string;
+    key: string;
+    relatedEntityId: string;
+    startTime: string;
+    endTime: string;
+}
+
+let mockRelationships: Relationship[] = [
```

Comment:
```suggestion
const mockRelationships: Relationship[] = [
```
mockRelationships is never reassigned. so its better to use 'const' instead of 'let'.
This is something related to lint errors.